### PR TITLE
add json file as mockup data

### PR DESCRIPTION
**Pull request by Abubakar-Meigag**
Create a JSON file for temporary data until we finish building our data structure; this file will be utilized.

### CODE REVIEW

File: client/src/modules-data.json [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+          "topicRef": "www.HTML-CSS.com"
+        },
+        {
+          "topic_id": 2,
+          "topicName": "Forms • Structuring Data",
+          "topicRef": "www.HTML-CSS.com"
+        },
+        {
+          "topic_id": 3,
+          "topicName": "Layout • Flex box • Grid",
```
Comment (ElenaBarker) [inline client/src/modules-data.json:19]: I will probably write "Flexbox" as one word.

File: client/src/modules-data.json [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+          "topicRef": "www.HTML-CSS.com"
+        },
+        {
+          "topic_id": 3,
```
Comment (ElenaBarker) [inline client/src/modules-data.json:23]: I guess you repeated topic 3 twice and missed topic 4. Can you fix that, please?

File: client/src/modules-data.json [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+          "topicName": "Layout • Flex box • Grid",
+          "topicRef": "www.HTML-CSS.com"
+        }
+      ]
+    },
+    {
+      "id": 2,
+      "module_name": "JavaScript_1",
```
Comment (ElenaBarker) [inline client/src/modules-data.json:31]: I think maybe a name exactly like in the CYF syllabus will be good to have, like "JavaScript_Core_1', following with the topic name "Hello World (of JS): variables, strings, types, numbers, statements, expressions, functions," etc.

**Review by ElenaBarker — APPROVED** [review]
Hi Beko, I think, if you will make these small changes that i write, the data will look better, so we can all use it. Thank you

File: client/src/modules-data.json [hunk 4]
```diff
@@ -0,0 +1,176 @@
+{
+  "modules": [
+    {
+      "id": 1,
+      "module_name": "Html-Css",
+      "topics": [
+        {
+          "topic_id": 1,
```
Comment (Abdirizak-Mohamed) [inline client/src/modules-data.json:8]: suggestion: we are using to naming conventions here. Can we try to pick one and enforce it for the entire project 
either 
camel case: e.g. `camelCase` 
underscore: e.g. `topic_id`

File: client/src/modules-data.json [hunk 5]
```diff
@@ -0,0 +1,176 @@
...
+          "topicRef": "www.JavaScript.com"
+        },
+        {
+          "topic_id": 4,
+          "topicName": "Array methods and higher order functions",
+          "topicRef": "www.JavaScript.com"
+        }
+      ]
+    },
+    {
```
Comment (Abdirizak-Mohamed) [inline client/src/modules-data.json:55]: Question: 
How do we plan to model this data in our database?

File: client/src/modules-data.json (hunk 1 above)
Comment (Abubakar-Meigag) [reply client/src/modules-data.json:19]: I have fixed the word 'Flexbox'. The issue has been resolved.

File: client/src/modules-data.json (hunk 2 above)
Comment (Abubakar-Meigag) [reply client/src/modules-data.json:23]: I have fixed the topic 4. The issue has been resolved.

File: client/src/modules-data.json (hunk 3 above)
Comment (Abubakar-Meigag) [reply client/src/modules-data.json:31]: I have fixed the topic name.

File: client/src/modules-data.json (hunk 4 above)
Comment (Abubakar-Meigag) [reply client/src/modules-data.json:8]: I have fixed the text and now it uses the camelCase convention throughout. The issue has been successfully resolved.

File: client/src/modules-data.json (hunk 5 above)
Comment (Abubakar-Meigag) [reply client/src/modules-data.json:55]: Done

**Pull request merged**